articles/openshift/oshift-how-configure-portworx-volume-replication.md

```diff
@@ -0,0 +1,139 @@
+    ```
+
+3. Excluding the IPs from the previous output will return the remaining node which the volume is not being replicated to:
+
+    ```none
+    $ oc exec $PX_POD -n kube-system -c portworx -- /opt/pwx/bin/pxctl cluster list | grep -vE '10.254.254.14|10.254.254.12'
+    Cluster ID: px-cluster-0db6c5ee-f3e9-4773-a863-3eb77d8d30ca
+    Cluster UUID: 48f080dd-56f2-4d4e-8df4-99cf5dbfd8a9
+    Status: OK
+    
+    Nodes in the cluster:
+    ID					SCHEDULER_NODE_NAME		DATA IP		CPU		MEM TOTAL	MEM FREE	CONTAINERS	VERSION			Kernel				OS						STATUS
+    6dcb56bc-9402-41ea-b819-56f2c9e1c742	worker-tenant-m-0.portworx-demo	10.254.254.13	2.402023	17 GB		15 GB		N/A		2.1.3.0-651d5d4		3.10.0-957.21.2.el7.x86_64	Red Hat Enterprise Linux Atomic Host 7.6.5	Online
+    ```
+
+4. You can then replicate the volume to the remaining node:
+
+    ```none
+    $ oc exec $PX_POD -n kube-system -c portworx -- /opt/pwx/bin/pxctl volume ha-update --repl=3 --node 6dcb56bc-9402-41ea-b819-56f2c9e1c742 pvc-e15d82b2-ed46-11e9-8422-fa163e52fd0e
+    Update Volume Replication: Replication update started successfully for volume pvc-e15d82b2-ed46-11e9-8422-fa163e52fd0e
+    ```
+
+5. Verify that the volume has been replicated to the additional node:
+
+    ```none
+    $ oc exec $PX_POD -n kube-system -c portworx -- /opt/pwx/bin/pxctl volume inspect pvc-e15d82b2-ed46-11e9-8422-fa163e52fd0e | grep -E 'HA|Node'
+            HA                       :  3
+                       Node          : 10.254.254.14 (Pool 0)
+                       Node          : 10.254.254.12 (Pool 0)
+                       Node          : 10.254.254.13 (Pool 0)
+    ```
+
+6. You can remove a replica from a node by specifying the node ID and reducing the replication factor:
+
+    ```none
+    $ oc exec $PX_POD -n kube-system -c portworx -- /opt/pwx/bin/pxctl volume ha-update --repl=2 --node 6dcb56bc-9402-41ea-b819-56f2c9e1c742 pvc-e15d82b2-ed46-11e9-8422-fa163e52fd0e
+    Update Volume Replication: Replication update started successfully for volume pvc-e15d82b2-ed46-11e9-8422-fa163e52fd0e
+    ```
+
+## Failover
+
+During a failover event (when using non-Portworx provisioned persistent volumes) the backing block storage device requires reattaching to the node where a pod is rescheduled. This can lead to increased recovery times, which Portworx aims to reduce through replication and a storage-aware scheduler that can schedule pods to nodes where a replica exists.
+
+### The STORK scheduler
+
+STORK (STorage Operator Runtime for Kubernetes) is an open source storage scheduler plugin that extends the default scheduler by exposing information regarding the capabilities and state of the underlying storage provider. Using a storage-aware scheduler improves scheduling decisions, resulting in improved performance and reduced recovery times for persistent volumes.
+
+To use the STORK scheduler, add the following entry to the pod spec:
+
+```none
+schedulerName: stork
+```
+
```

Comment:
This feels a little out of place down here on its own, should we elaborate further on it (maybe even in a separate article if needed)?

Reply:
I don't think there is enough content to warrant writing a dedicated article for this. The reason for this being included in the same article was due to the STORK scheduler being a requirement if you use a replication factor > 1 as the default scheduler could schedule a pod to a node that doesn't have a replicated volume.